lib/SIL/LinearLifetimeChecker.cpp

```diff
@@ -66,12 +71,13 @@ struct State {
 
   /// A list of successor blocks that we must visit by the time the algorithm
   /// terminates.
-  SmallPtrSet<SILBasicBlock *, 8> successorBlocksThatMustBeVisited;
+  SmallSetVector<SILBasicBlock *, 8> successorBlocksThatMustBeVisited;
```

Comment:
Just an FYI, the reason that I made this change here is that I want to make sure that we do not run into non-determinism issues by iterating over a pure set. Instead here we get the insertion order traversal... the sweet smell of determinism!